Sources/Transaction.swift

```diff
@@ -10,13 +10,40 @@ import BigInt
 public struct Transaction {
     public var accountNonce: UInt64
     public var price: BigInt
-    public var gasLimit: BigInt
-    public var recipient: Address?
+    public var gasLimit: UInt64
+    public var recipient: Address
```

Comment:
can we change `recipient` => `to` for consitency?

Reply:
I believe it could be optional

Reply:
I based it on the geth implementation, I think it's more important to be consistent with that: https://github.com/ethereum/go-ethereum/blob/ba1030b6b84f810c04a82221a1b1c0a3dbf499a8/core/types/transaction.go